test/test_kernel_launch_checks.py

```diff
@@ -0,0 +1,38 @@
+from torch.testing._internal.common_utils import TestCase
```

Comment:
Could you modify this line to also import `run_tests`, and then also add the following to the bottom of the file?
```python
if __name__ == '__main__':
    run_tests()
```
This would allow your new test file to be run using `python test/test_kernel_launch_checks.py` similar to existing `test/*.py` files.

Reply:
Fixed.